src/app/api/services/dot-interceptor/dot-interceptor.service.ts

```diff
@@ -0,0 +1,87 @@
+import { DotMessageService } from './../dot-messages-service';
```

Comment:
This is the new "interceptor" service, there is only one public method that we should start using for I think everything, we need to talk about more about this.

Reply:
Yes we should start using this everywhere

Reply:
I'm going to use it in all edit content and layout and content type.